src/constants/earthEngineLayers/population_total_WorldPop.js

```diff
@@ -0,0 +1,41 @@
+import i18n from '@dhis2/d2-i18n'
+import { EARTH_ENGINE_LAYER } from '../layers.js'
+
+export default {
+    layer: EARTH_ENGINE_LAYER,
+    layerId: 'WorldPop/GP/100m/pop_age_sex_cons_unadj_TOTAL',
+    img: 'images/population.png',
+    datasetId: 'WorldPop/GP/100m/pop_age_sex_cons_unadj',
+    format: 'ImageCollection',
+    name: i18n.t('Population'),
+    description: i18n.t('Estimated number of people living in an area.'),
+    source: 'WorldPop / Google Earth Engine',
+    sourceUrl:
+        'https://developers.google.com/earth-engine/datasets/catalog/WorldPop_GP_100m_pop_age_sex_cons_unadj',
+    unit: i18n.t('people per hectare'),
+    defaultAggregations: ['sum', 'mean'],
+    periodType: 'YEARLY',
+    useCentroid: true,
```

Comment:
We should double check weighted vs unweighted (ie useCentroid: true).

Reply:
We agreed on keeping the logic in case it becomes relevant in the future but not use it.